tests/contrib/test_django_orm.py

```diff
@@ -38,6 +38,9 @@
 django.setup()
 from django.apps import AppConfig
 
+import mock
+from mock import MagicMock
```

Comment:
Why not just use `mock.MagicMock` below? Also, as @jonparrott taught me, only `mock.Mock` is needed in most situations, unless you need magic methods also mocked.